docker/Dockerfile

```diff
@@ -1,75 +1,61 @@
-FROM ubuntu:16.04
-LABEL maintainer "Alexander Panin <justheuristic@gmail.com>, Dmitry Mittov <mittov@gmail.com>"
+ARG device
+
+FROM ubuntu:18.04 AS base-cpu
+
+FROM nvidia/cuda:10.2-base-ubuntu18.04 AS base-gpu
+
+FROM base-$device AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update && \
+    apt-get install -y wget unzip git cmake xvfb sudo freeglut3-dev ffmpeg
+
+RUN adduser --disabled-password --gecos "Default user" jupyter && \
```

Comment:
Not important by any measure, but would be nice to have this: could you make the username a build arg, like so

```
ARG USERNAME=jovyan
```

replacing `jupyter` with `jovyan`, like in [Jupyter Docker images](https://github.com/jupyter/docker-stacks/issues/358#issuecomment-288844834)?

As far as I see, after making that change, a simple `s/jupyter/$USERNAME/g` will fix the rest of the Dockerfile.

Reply:
fixed, check and resolve